CONTRIBUTING.md

```diff
@@ -6,12 +6,12 @@ Thank you!
 
 ## Legal Info
 
-When you open your first pull-request to ClickHouse repo, a bot will invite you to accept ClickHouse Individual CLA (Contributor License Agreement). It is a simple few click process. For subsequent pull-requests the bot will check if you have already signed it and won't bother you again.
+When you create your first pull request to the ClickHouse repository, a bot will invite you to accept the ClickHouse Individual CLA (Contributor License Agreement). It is a simple, few-click process. For subsequent pull requests, the bot will check if you have already signed them and won't bother you again.
```

Comment:
> For subsequent pull requests, the bot will check if you have already signed them

This is wrong.

Not "them" (pull requests),
but "it" or "that" (Contributor License Agreement).